packages/parser/lib/Lexer.re

```diff
@@ -542,22 +532,28 @@ let handle_tokenizer_error = lexbuf => {
 };
 
 let skip_whitespace = ref(false);
+let prev_skip_whitespace = ref(false);
```

Comment:
whoa, I would need an explanation on why previous is needed, when we have skip_ws

Reply:
If both booleans are needed, I would go with a single ref and a few variants, wdyt?

Inside_media | Inside_selector | Outside_x | etc..

Reply:
oof forgot to refactor this specific hack hehe.
when we are inside a brace, the skip_whitespace is set to true and the whitespace inside values (<property>: <values>) is skipped. i want to enable whitespace when we are seeing `:` and bring skip_whitespace to its previous value when we are seeing `;`

i like the variant idea though, will try to implement it!